.github/ISSUE_TEMPLATE/new-story-template.yml

```diff
@@ -0,0 +1,71 @@
+name: Modernisation platform Story
+description: 📖 Create new Modernisation platform DevOps story
+title: "[gitmoji] <title>"
+labels: ["needs refining"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out the fields below.
+  - type: textarea
+    id: user-story
+    attributes:
+      label: User Story
+      placeholder: "As a… [who is the user?] 
+              I need/want/expect to… [what does the user want to do?]
+              So that… [why does the user want to do this?] "
+      value: "As a…  
+
+              I need/want/expect to… 
+
+              So that…"
+    validations:
+      required: true
+  - type: textarea
+    id: value-purpose
+    attributes:
+      description: Describe the value and purpose of the ticket.
+      label: Value / Purpose
+      value:
+    validations:
+      required: false
+  - type: input
+    id: useful-contacts
+    attributes:
+      description: Please add any useful contacts, these may include; Stakeholders, SME’s or 3rd Parties.
+      placeholder: Joe Bloggs
+      label: Useful Contacts
+      value:
+    validations:
+      required: false
+  - type: textarea
+    id: additional-information
+    attributes:
+      description: Please add any useful links or additional information that would be beneficial to anyone working on this Issue.
+      label: Additional Information 
+      value:
+    validations:
+      required: false
+  - type: textarea
+    id: dod
+    attributes:
+      description: Please clearly and concisely detail the DoD. (example optional checklist below)
```

Comment:
Personally I'd just write `Definition of Done` here as it's not previously described and shown as an acronym.